app/Models/PortVlan.php

```diff
@@ -7,4 +7,11 @@ class PortVlan extends PortRelatedModel
     protected $table = 'ports_vlans';
     protected $primaryKey = 'port_vlan_id';
     public $timestamps = false;
+
+    // ---- Define Relationships ----
+
+    public function vlan1()
```

Comment:
Fix me, collides with column `vlan` else

Reply:
This relationship isn't quite right because there will be multiple vlans with the same number. (for each device).
so it would be vlans() anyway.